.github/workflows/create-demo-clusters.yml

```diff
@@ -344,6 +398,103 @@ jobs:
           ./scale/launch_workload.sh np-load
           echo "Fake workload has been deployed to the long-running cluster" >> "$GITHUB_STEP_SUMMARY"
 
+  start-secured-cluster:
+    name: Start secured cluster
+    needs: [properties, start-acs, create-long-running-secured-cluster]
+    runs-on: ubuntu-latest
+    env:
+      SECURED_CLUSTER_NAME: ${{ inputs.secured-cluster-name }}
+      TAG: ${{github.event.inputs.version}}
+      KUBECONFIG: artifacts/kubeconfig
+      INFRA_TOKEN: ${{secrets.INFRA_TOKEN}}
+      USE_GKE_GCLOUD_AUTH_PLUGIN: "True"
+      ROX_ADMIN_PASSWORD: ${{ needs.start-acs.outputs.rox_password }}
+      ROX_ADMIN_USERNAME: admin
+      CENTRAL_IP: ${{ needs.start-acs.outputs.central-ip }}
+    steps:
+      - uses: stackrox/actions/infra/install-infractl@v1
+      - name: Test readiness for central
+        run: |
+          STATUS=$(infractl get "${SECURED_CLUSTER_NAME//./-}" --json | jq -r .Status)
+          if [ "$STATUS" != "READY" ]; then
+            exit 1
+          fi
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          ref: jv-ROX-17741-long-running-cluster-should-include-collector
+          repository: stackrox/stackrox
+      - uses: "google-github-actions/auth@v1"
+        with:
+          credentials_json: "${{ secrets.GCP_RELEASE_AUTOMATION_SA }}"
+      - name: "Set up Cloud SDK"
+        uses: "google-github-actions/setup-gcloud@v1"
+        with:
+          install_components: "gke-gcloud-auth-plugin"
+      - name: Download artifacts
+        id: artifacts
+        run: |
+          infractl artifacts "${SECURED_CLUSTER_NAME//./-}" -d artifacts
+      - name: Launch secured cluster
+        id: launch-secured-cluster
+        env:
+          MAIN_IMAGE_TAG: ${{github.event.inputs.version}} # Release version, e.g. 3.63.0-rc.2.
+          PAGERDUTY_INTEGRATION_KEY: ${{ secrets.RELEASE_MANAGEMENT_PAGERDUTY_INTEGRATION_KEY }}
+          REGISTRY_USERNAME: ${{ secrets.QUAY_RHACS_ENG_RO_USERNAME }}
+          REGISTRY_PASSWORD: ${{ secrets.QUAY_RHACS_ENG_RO_PASSWORD }}
+          STACKROX_DIR: ${{ github.workspace }}
+          #local_script_url: /repos/${{github.repository}}/contents/.github/workflows/scripts/start-secured-cluster.sh?ref=${{ github.ref_name }}
+          local_script_url: /repos/stackrox/actions/contents/.github/workflows/scripts/start-secured-cluster.sh?ref=jv-ROX-17741-long-running-cluster-should-include-collector
+        run: |
+          gh api -H "$ACCEPT_RAW" "$local_script_url" | bash -s
```

Comment:
Consider pasting the contents of the script here if it is called only from one place.
If that script is used in other places, then let's make it an action.

Reply:
Done